website_portal_v10/controllers/main.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+from openerp import http
+from openerp.http import request
+from openerp import tools
+from openerp.tools.translate import _
+
+from openerp.fields import Date
+
+
+class website_account(http.Controller):
```

Comment:
Although a backport, we should follow OCA guidelines. Merge won't be possible anyway because there are a lot of things that diverges soon (that were the conclusions when backporting account_bank_statement_import module).